pcsx2/gui/Panels/BiosSelectorPanel.cpp

```diff
@@ -115,6 +118,8 @@ Panels::BiosSelectorPanel::BiosSelectorPanel( wxWindow* parent )
 	*this	+= 8;
 	*this	+= m_FolderPicker	| StdExpand();
 
+	Bind(pxEvt_BiosEnumerationFinished, &BiosSelectorPanel::OnEnumComplete, this);
+
```

Comment:
Why the linefeed?

Reply:
Usually, there is a linefeed in between button click binds and all other binds. There are only two here so I could remove it if you'd like.

Reply:
Eh, I don't have any strong thoughts on that one, do as you prefer